pages/sending/raw.rst

```diff
@@ -0,0 +1,11 @@
+********************
+Ingest Raw/Plaintext
+********************
+
+The build-in ``RAW/Plaintext`` input is a netcat like application in Graylog. It will receive any data the is ingest to the running input. 
```

Comment:
```suggestion
The built-in ``RAW/Plaintext`` input is a netcat like application in Graylog. It will receive any data that is ingested into the running input. 
```